pyanaconda/ui/common.py

```diff
@@ -480,6 +480,8 @@ class StandaloneSpoke(Spoke):
     preForHub = None
     postForHub = None
 
+    priority = 100
```

Comment:
It looks like the default priority should be zero.

Reply:
This is default priority from the past but we can change it to make it stable with the current state.